### PR TITLE
Add galaxyproject/SARS-CoV-2 link and reference.

### PR DESCRIPTION
**Pull request by heuermh**
# nf-core/covid19 pull request

Many thanks for contributing to nf-core/covid19!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/covid branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/covid19)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/covid19/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,17 @@ The nf-core/covid19 pipeline comes with documentation about the pipeline, found
 
 nf-core/covid19 was originally written by Michael L Heuer, Jordan Wilheim, Maxime Garcia, Alexander Peltzer.
 
+Initial workflow development will be based on workflows developed in:
+
+Initial analysis of COVID-19 data using Galaxy, BioConda and public research infrastructure (XSEDE, de.NBI-cloud, ARDC cloud)
```
Comment (drpatelh) [inline README.md]:
```suggestion
Initial workflow development will be based on: [Initial analysis of COVID-19 data using Galaxy, BioConda and public research infrastructure (XSEDE, de.NBI-cloud, ARDC cloud)](https://github.com/galaxyproject/SARS-CoV-2)
```

File: README.md [hunk 2]
```diff
@@ -56,6 +56,17 @@ The nf-core/covid19 pipeline comes with documentation about the pipeline, found
 
 nf-core/covid19 was originally written by Michael L Heuer, Jordan Wilheim, Maxime Garcia, Alexander Peltzer.
 
+Initial workflow development will be based on workflows developed in:
```
Comment (drpatelh) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -56,6 +56,17 @@ The nf-core/covid19 pipeline comes with documentation about the pipeline, found
...
+
+Initial analysis of COVID-19 data using Galaxy, BioConda and public research infrastructure (XSEDE, de.NBI-cloud, ARDC cloud)
+https://github.com/galaxyproject/SARS-CoV-2
```
Comment (drpatelh) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (drpatelh) [reply README.md]: Maybe move this to another section called `## Pipeline summary` after `## Introduction` like here:
https://github.com/nf-core/atacseq#pipeline-summary

We can then use it as a placeholder and fill in the blanks once we know exactly what the implementation will look like.

**Comment by heuermh** [comment]
Thanks @drpatelh!  The latest commit should address your review comments.  Feel free to squash commits at merge or ask me to rebase + force push.

**Pull request merged**